res/mac/FreenetTray/src/defaults.plist

```diff
@@ -2,8 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>firstlaunch</key>
+	<true/>
 	<key>nodepath</key>
 	<string>$INSTALL_PATH</string>
+	<key>nodefcpurl</key>
+	<string>tcp://127.0.0.1:9481</string>
```

Comment:
Oh, and I could in fact add a UI setting at some point for those ports and the address so the user could change it without having to mess with the `defaults` command or editing plists.